src/resolve.rs

```diff
@@ -54,6 +54,9 @@ impl Resolver {
 
     #[tracing::instrument]
     pub(crate) fn resolve_as_file(&self, info: Info) -> State {
+        if info.request.target.ends_with('/') || info.request.target.ends_with('\\') {
```

Comment:
@bvanjoi is this legit? Let it abort if the requested target ends with a slash.

Reply:
Maybe not, `require('./a/')` might resolve to `<prefix>/a/index.js`

Reply:
https://github.com/modern-js-dev/nodejs_resolver/pull/123 this pr may impl this